metadata_service/proxy/atlas_proxy.py

```diff
@@ -39,9 +39,10 @@ class AtlasProxy(BaseProxy):
     TABLE_ENTITY = app.config['ATLAS_TABLE_ENTITY']
     DB_ATTRIBUTE = app.config['ATLAS_DB_ATTRIBUTE']
     STATISTICS_FORMAT_SPEC = app.config['STATISTICS_FORMAT_SPEC']
-    READER_TYPE = 'Reader'
+    BOOKMARK_TYPE = 'Bookmark'
+    USER_TYPE = 'User'
     QN_KEY = 'qualifiedName'
-    BKMARKS_KEY = 'isFollowing'
+    BOOKMARK_ACTIVE_KEY = 'active'
```

Comment:
Do you need to make this a constant ? not adding any extra value, just making things complex to read.

Reply:
it's used in 4 different places throughout this class, so for consistency it made sense to define it as constant